packages/core/content-manager/admin/src/history/components/VersionInputRenderer.tsx

```diff
@@ -328,31 +333,25 @@ const VersionInputRenderer = ({
     case 'blocks':
       return <BlocksInput {...props} hint={hint} type={props.type} disabled={fieldIsDisabled} />;
     case 'component':
-      return (
-        <ComponentInput
-          {...props}
-          hint={hint}
-          disabled={fieldIsDisabled}
-          renderLayout={(layoutProps) => {
-            // Components can only have one panel, so only save the first layout item
-            const [remainingFieldsLayout] = getRemaingFieldsLayout({
-              layout: [layoutProps.layout],
-              metadatas: configuration.components[props.attribute.component].metadatas,
-              fieldSizes,
-              schemaAttributes: components[props.attribute.component].attributes,
-            });
+      const { layout } = compos[props.attribute.component];
 
-            return (
-              <ComponentLayout
-                {...layoutProps}
-                layout={[...layoutProps.layout, ...remainingFieldsLayout]}
-                renderInput={(inputProps) => (
-                  <VersionInputRenderer {...inputProps} shouldIgnoreRBAC={true} />
-                )}
-              />
-            );
-          }}
-        />
+      const [remainingFieldsLayout] = getRemaingFieldsLayout({
+        layout: [layout],
+        metadatas: configuration.components[props.attribute.component].metadatas,
+        fieldSizes,
+        schemaAttributes: components[props.attribute.component].attributes,
+      });
+
+      return (
+        <ComponentInput {...props} hint={hint} disabled={fieldIsDisabled}>
+          <ComponentLayout
+            name={props.name}
+            layout={layout}
```

Comment:
ℹ️ just passing layout here for now since I'm not sure if / how it could work with the remainingFieldsLayout function...